src/app/components/Upload.svelte

```diff
@@ -14,26 +14,28 @@
   // This problem can also be solved by listening for dragOver
   // However, as of right now, doing so on Chrome when dragging over nested element
   // will cause the drag over effect to be cancelled for a short amount of time, then resume
+  let files: File[] = [];
   let dragEnterCounter = 0;
   let error: Error | null = null;
 
   async function uploadFilesFromDragAndDrop(event: DragEvent) {
     dragEnterCounter = 0;
-    let fileList = filesFromDragEvent(event);
-    await uploadAllFilesOrDisplayError(fileList);
+    const file = filesFromDragEvent(event);
+    await uploadAllFilesOrDisplayError(file);
   }
 
   async function uploadFilesFromInputElement(event: Event) {
-    const files = filesFromInputElement(event.target);
-    await uploadAllFilesOrDisplayError(files);
+    const file = filesFromInputElement(event.target);
+    await uploadAllFilesOrDisplayError(file);
   }
 
-  async function uploadAllFilesOrDisplayError(files: File[]): Promise<void> {
-    if (files.length === 0) return;
+  async function uploadAllFilesOrDisplayError(file: File[]): Promise<void> {
```

Comment:
Could we name `file` here to `uploadedFiles` (or something similar)? It might be more clear to have `uploadedFiles` and `files` rather than `file` and `files`